src/libstore/build.cc

```diff
@@ -1744,21 +1735,10 @@ void DerivationGoal::startBuilder()
 
     if (useChroot) {
 
-        string defaultChrootDirs;
-#if __linux__
-        if (worker.store.isInStore(BASH_PATH))
```

Comment:
I got rid of this because I can't check `isInStore` from the `Settings` class. How likely is this to be false? It seems like if bash is missing, it'd break a ton of stuff...